dali/python/nvidia/dali/auto_aug/core/_augmentation.py

```diff
@@ -358,9 +358,9 @@ def _get_param(self, magnitude_bin, num_magnitude_bins, param_device):
             return None
         if magnitude_bin is None:
             raise Exception(
-                f"The augmentation `{self.name}` has `mag_range` specified, so when called, "
-                f"it requires `magnitude_bin` parameter to select the magnitude from the "
-                f"`mag_range`.\nError in augmentation: {self}."
+                f"The augmentation `{self.name}` has `mag_range` specified, "  # nosec B608
+                f" so when called, it requires `magnitude_bin` parameter to select "
```

Comment:
nitpick: there's double space between `specified,` and `so when`.

Reply:
Fixed